tests/unit/BucketTest.php

```diff
@@ -20,7 +20,10 @@ public function newClient()
 	{
 		$api_key = getenv('API_KEY');
 		$reference_id = getenv('REFERENCE_ID');
-		$this->client = new  \Supabase\Storage\StorageClient($api_key, $reference_id);
+		$scheme = 'https';
```

Comment:
After the __constructor is updated none of these should be needed - please remove them.